docs/documentation/server_admin/topics/organizations/managing-identity-providers.adoc

```diff
@@ -0,0 +1,42 @@
+[id="managing-organization-identity-providers_{context}"]
+
+[[_managing_identity_provider_]]
+=  Managing identity providers
+[role="_abstract"]
+An organization might have its own identity provider as the single source of truth for their identities. In this case,
+you want to configure the organization to authenticate users using the organization's identity provider, federate their
+identities, and finally add them as a member of the organization.
+
+An organization can have one or more identity providers associated with it so that they can authenticate their users from
+different sources and enforce different constraints on each of them.
+
+The first step before linking an identity provider to an organization is to create one at the realm level from the `Identity Providers`
+section in the menu. You can link any of the built-in social and identity providers available in the realm to an organization.
+
+An identity provider can be linked to an organization from the `Identity providers` tab.
```

Comment:
Is this a repetittion of the preceding sentence or something new?

Reply:
It is not, that one is about the `Identity Providers` tab.